lib/actions/base.js

```diff
@@ -39,9 +39,11 @@ Base.prototype.display = function (data) {
   var opts = this.opts;
   var format = opts.format;
 
-  if (format === "json") {
+  if (format === "object") {
+    return data;
+  } else if (format === "json") {
     return JSON.stringify(data, null, JSON_SPACES);
-  } else if (format === "text") {
+  } if (format === "text") {
```

Comment:
Replaced with a switch statement 😄 Surprised that I couldn't find a lint rule that covers this, though. It's pretty devious!